style.css

```diff
@@ -769,7 +803,7 @@ textarea::placeholder {
     display: flex;
   }
 
-  .sidep img {
+  /* .sidep img {
     content: url("./Work/1.png");
   }
 

```

Comment:
- [ ] It is considered best practice to avoid keeping commented codes or unused codes in your work base, as it makes it messy. Kindly remove this unused part of your code. :wrench: